throughput/helper.go

```diff
@@ -37,4 +37,7 @@ type SpamHelper interface {
 	// Memo is used to ensure that each transaction is unique (even if between the same
 	// sender and receiver for the same amount).
 	GetTransfer(address codec.Address, amount uint64, memo []byte) []chain.Action
+
+	// GetActions returns a list of actions the spammer sends to the network.
+	GetActions() []chain.Action
```

Comment:
I feel weird about adding this to the interface... I would feel better if we just returned a list of keys that this created (i.e sh.pks) back to the caller so they could figure out how to create the transactions via `GetTransfer`

Reply:
we want to decouple `GetTransfer` from the caller(spam script). 
For a little more context, the VM implements this helper interface. The point of this PR is to allow the spam script to generate txs with arbitrary actions not just `transfer`. It just so happens that  the action we want to test for `morpheus` vm in `GetActions` is the same as `GetTransfer`